### DISCUSSION TITLE
Is this repo only for GSSOC ?

### DISCUSSION THREAD

**Discussion by NanoTeddy** [Akshatchaube01/TimeWarp#402] [Q&A] [answered]
Will I able to contribute after 10 Auguts ?

**Comment by MeetDOD** [comment, marked as answer]
Ya you can contribute to this repo because this is open source but the GSSOC'24 end on 10 you will not get points, but you can contribute for your learning and experience in open source 👍

**Reply by NanoTeddy** [reply]
Ok got it 👍
